NPM package misses index.js

```
Error: Cannot find module '…/recommender-playground/node_modules/@vscode/vscode-extension-recommender/dist/index.js'. Please verify that the package.json has a valid "main" entry
```

Looks like the `dist/` folder only contains `index.d.ts`, no `lib/` folder.

--- a/.npmignore
+++ b/.npmignore
@@ -1,6 +1,8 @@
 build
 test
 .vscode
+.github
+cli
 lib
 tsconfig.json
 webpack.config.js

--- a/package.json
+++ b/package.json
@@ -1,12 +1,9 @@
 {
   "name": "@vscode/vscode-extension-recommender",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Deep learning model to recommend the best extension for a given VS Code workspace",
-  "main": "dist/index.js",
-  "bin": {
-    "vscode-extension-recommender": "cli/index.js"
-  },
-  "module": "dist/index.js",
+  "main": "dist/lib/index.js",
+  "module": "dist/lib/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "prepublishOnly": "npm test && npm run build",

